portfolio-page.html

```diff
@@ -0,0 +1,166 @@
+						alt="A picture for project-2">
+					<figcaption class="project-caption">Project-2</figcaption>
+				</figure>
+				<p class="project-info">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus est mauris, aliquam in felis a, ullamcorper ornare est. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Cras vulputate ligula neque, nec faucibus lacus auctor nec. Sed accumsan fringilla neque, vel mollis ligula interdum quis. Sed turpis mi, volutpat at pretium id, porttitor eget nunc. Fusce convallis dolor bibendum finibus finibus. Pellentesque rutrum lobortis ex eu fermentum.</p>
+			</article>
+
+			<article class="project-content">
+				<h3 class="project-title">Project-3</h3>
+				<figure class="project-figure">
+					<img class="project-image" src="images/terminal-wizard.jpeg" 
+						alt="A picture for project-3">
+					<figcaption class="project-caption">Project-3</figcaption>
+				</figure>
+				<p class="project-info">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus est mauris, aliquam in felis a, ullamcorper ornare est. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Cras vulputate ligula neque, nec faucibus lacus auctor nec. Sed accumsan fringilla neque, vel mollis ligula interdum quis. Sed turpis mi, volutpat at pretium id, porttitor eget nunc. Fusce convallis dolor bibendum finibus finibus. Pellentesque rutrum lobortis ex eu fermentum.</p>
+			</article>
+
+			<article class="project-content">
+				<h3 class="project-title">Project-4</h3>
+				<figure class="project-figure">
+					<img class="project-image" src="images/terminal-wizard.jpeg" 
+						alt="A picture for project-4">
+					<figcaption class="project-caption">Project-4</figcaption>
+				</figure>
+				<p class="project-info">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus est mauris, aliquam in felis a, ullamcorper ornare est. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Cras vulputate ligula neque, nec faucibus lacus auctor nec. Sed accumsan fringilla neque, vel mollis ligula interdum quis. Sed turpis mi, volutpat at pretium id, porttitor eget nunc. Fusce convallis dolor bibendum finibus finibus. Pellentesque rutrum lobortis ex eu fermentum.</p>
+			</article>
+		</section> <!-- .portfolio-section -->
+
+		<section class="cheat-sheets-section" id="cheat-sheets">
```

Comment:
Cheat sheets are cool but it's hard to tell what they are because there is no heading or anything, also the link at the bottom of each one doesn't indicate that it takes to Notion, my recommendation is to either:
- add a heading over the section that says "Cheat sheets" or something like that
- in each heading, change to e.g. HTML Cheat sheet rather than just HTML
also:
- instead of Link put Link to Notion or Notion link (this one is less important)

this is purely a UX thing, I think it would make it clearer what I am looking at